includes/class-freemius.php

```diff
@@ -15003,6 +14994,14 @@ private function embed_submenu_items() {
                             array( $this, '' )
                         );
                     }
+
+                    if ( $item['show_submenu'] && ! $removed_first_submenu_item ) {

```

Comment:
@fajardoleo please add that Please add the explanation that we just discussed on slack including a “visual” example of what happens:
```
    Awesome Plugin
        - Awesome Plugin  <-- we want to remove this since there's no real setting page for the top-level
```